app/controllers/users_controller.rb

```diff
@@ -1,8 +1,5 @@
 class UsersController < ApplicationController
-
-
-
   def show
-    @user = User.find(params[:id])
+    @user = current_user
```

Comment:
Is this necessarily true? If I'm user 9, and I go to `/users/10` do I really want to see my profile?